src/Poseidon/CLI/Init.hs

```diff
@@ -1,10 +1,9 @@
 module Poseidon.CLI.Init where
 
 import           Poseidon.BibFile           (dummyBibEntry, writeBibTeXFile)
-import           Poseidon.GenotypeData      (GenotypeDataSpec (..),
-                                             GenotypeFormatSpec (..), 
+import           Poseidon.GenotypeData      (InGenotypeData (..),
```

Comment:
See my other comment... I think these are redundant data types. They should be merged.

Reply:
Let's discuss this above.